src/qibocal/protocols/characterization/utils.py

```diff
@@ -373,3 +376,61 @@ def significant_digit(number: float):
         position = max(position, np.ceil(-np.log10(abs(np.imag(number)))))
 
     return int(position)
+
+
+def table_dict(
+    qubit: Union[list[QubitId], QubitId],
+    names: list[str],
+    values: list,
+    display_error=False,
+) -> dict:
+    """
+    Build a dictionary to generate HTML table with `table_html`.
+
+    Args:
+        qubit (Union[list[QubitId], QubitId]): If qubit is a scalar value,
+        the "Qubit" entries will have only this value repeated.
+        names (list[str]): List of the names of the parameters.
+        values (list): List of the values of the parameters.
+        display_errors (bool): if `True`, it means that `values` is a list of value-error couples,
+        so an `Errors` key will be displayed in the dictionary. The function will round the couples according to their significant digits. Default False.
+
+    Return:
+        A dictionary with keys `Qubit`, `Parameters`, `Values` (`Errors`).
+    """
+    if not np.isscalar(values):
+        if np.isscalar(qubit):
+            qubit = [qubit] * len(names)
+
+        if display_error:
+            rounded_values, rounded_errors = round_report(values)
+
+            return {
+                "Qubit": qubit,
+                "Parameters": names,
+                "Values": rounded_values,
+                "Errors": rounded_errors,
+            }
+    else:  # If `values` is scalar also `qubit` should be a scalar
+        qubit = [
+            qubit
+        ]  # In this way when the Dataframe is generated, an index is not required.
+    return {"Qubit": qubit, "Parameters": names, "Values": values}
+
+
+def table_html(data: dict) -> str:
+    """This function converts a dictionary into an HTML table.
+
+    Args:
+        data (dict): the keys will be converted into table entries and the
+        values will be the columns of the table.
+
+    Return:
+        str
+    """
+    fitting_report = pd.DataFrame(data)
+    fitting_report = fitting_report.to_html(
+        classes="fitting-table", index=False, border=0
+    )
```

Comment:
```suggestion
    fitting_report = pd.DataFrame(data).to_html(
        classes="fitting-table", index=False, border=0
    )
```
Not much to save characters, but to avoid shadowing.